packages/amplify-cli-utils/src/index.js

```diff
@@ -214,6 +214,47 @@ export function hlVer(toVer, fromVer) {
 	return (toMatch && toMatch[1] || '') + version.join('.') + tag();
 }
 
+/**
+ * Initializes the Amplify SDK, loads an account, and finds the default org id.
+ *
+ * @param {String} [accountName] - The name of the platform account to use.
+ * @param {String} [org] - The name, id, or guid of the default organization.
+ * @returns {Promise<Object>}
+ */
+export async function initPlatformAccount(accountName, org) {
+	const { config, sdk } = initSDK();
+	const account = await sdk.auth.find(accountName || config.get('auth.defaultAccount'));
+
+	if (accountName) {
+		if (!account) {
+			throw new Error(`Account "${accountName}" not found`);
+		} else if (!account.isPlatform) {
+			throw new Error(`Account "${accountName}" is not a platform account\n\nTo login, run: axway auth login`);
+		}
+	} else if (!account || !account.isPlatform) {
+		throw new Error('You must be logged into a platform account\n\nTo login, run: axway auth login');
+	}
+
+	if (org) {
+		org = await sdk.org.find(account, org);
+	} else {
+		try {
+			// check the config for a default org for this account
+			org = await sdk.org.find(account, config.get(`auth.defaultOrg.${account.hash}`));
```

Comment:
Maybe can be combined in one try-catch block without if-else here (same as you do with the account)?
```
try {
  org = await sdk.org.find(account, config.get(org || `auth.defaultOrg.${account.hash}`));
} catch (err) {
...blabla
}
```

Reply:
I tried rewriting it a couple ways and there's really no way to clean this up. There are 3 scenarios this code is supporting:

1. You passed in `--org <name>` and if it doesn't exist, we do not want to catch the error, but instead let the error handler catch it and exit out.
2. We load the default org from the config file. The default org value exists and is valid.
3. The default org in the config file does not exist or is not found (or is stale) and the we should squelch the error and pick the org that platform.axway.com says is the "default".

Reply:
Looks good to me, it is a minor suggestion anyway )